include/NovelRT/Timing/DeltaTimestamp.h

```diff
@@ -0,0 +1,34 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_TIMING_DELTATIMESTAMP_H
+#define NOVELRT_TIMING_DELTATIMESTAMP_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Timing {
+  class DeltaTimestamp {
```

Comment:
nit: This should probably be called just `Timestamp` or `TimeSpan` or something.

It doesn't strictly speaking have to be a delta, it just represents some amount of time in 100ns ticks.